components/autofill/src/sync/credit_card/outgoing.rs

```diff
@@ -20,12 +20,27 @@ pub(super) struct OutgoingCreditCardsImpl {
     pub(super) encdec: EncryptorDecryptor,
 }
 
+// impl OutgoingCreditCardsImpl {
```

Comment:
This was my original change. As expected it still failed because the issue is that the local record doesn't have an associated mirror record.